docs/DeveloperGuide.md

```diff
@@ -1,38 +1,527 @@
+| v1.0    | Hawker Store Owner | Delete an item                                   | Remove it from the menu                                                          |
+| v1.0    | Hawker Store Owner | List all items                                   | View all items in the menu, including its index and price                        |
+| v1.0    | Hawker Store Owner | Add an item to an order                          | Track what customers order and calculate its total cost                          |
+| v1.0    | Hawker Store Owner | Add multiple items to an order                   | Save time by combining multiple items from a single customer into one order      |
+| v1.0    | Hawker Store Owner | List orders                                      | See all orders, including the total cost of each other                           |
+| v1.0    | Returning User     | Save menu items                                  | Save time by using the same menu, without entering each item over again each day |
+| v1.0    | Returning User     | Save transactions from previous days             | Keep track of all orders easily, even across multiple days                       |
+| v1.1    | Hawker Store Owner | Search for an item                               | Check its index or price                                                         |
+| v1.1    | Hawker Store Owner | Add items to an order by name                    | Add items without remembering its index                                          |
+| v2.0    | Hawker Store Owner | Void an order                                    | Provide refunds or ignore orders with wrong inputs                               |
+| v2.0    | New User           | Add an item step-by-step                         | Get used to the new system                                                       |
+| v2.0    | New user           | Delete an item step-by-step                      | Get used to the new system                                                       |
+| v2.0    | New User           | Add an item(s) to an order step-by-step          | Get used to the new system                                                       |
+| v2.0    | Returning User     | See statistics on based on previous transactions | **TBC**                                                                          |
 
-* *glossary item* - Definition
+### Non-functional Requirements
 
-## Instructions for manual testing
+1. The application should be able to run on any operating system with `Java 11` installed.
+2. The application should be responsive.
+3. The application should be simple enough for a novice who is not familiar with a Command Line Interface (CLI) to use.
+4. The application should be easily adaptable to people who are already well-versed in using a traditional
+   Point-of-Sale (POS) system.
 
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+### Glossary
```

Comment:
I find that there are many things that have not finished yet. Hopefully you won't forget this part! Overall, that's a good job!👍